stable/mysql/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: mysql
-version: 0.10.2
+version: 0.10.3
```

Comment:
Could you bump the minor version as this introduces a new feature?

Reply:
0.11.0?

Reply:
Yep